CLR/Libraries/Windows_Devices/windows_devices_stub.cpp

```diff
@@ -0,0 +1,61 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include "windows_devices.h"
+
+HRESULT Library_windows_devices_native_Windows_Devices_Gpio_GpioController::get_PinCount___I4( CLR_RT_StackFrame& stack )
+{
+    TINYCLR_HEADER();
+    TINYCLR_SET_AND_LEAVE(stack.NotImplementedStub());

```

Comment:
This is copy-pasted from the SPOT stubs.  I'll change this to CLR_E_NOT_SUPPORTED, though it does mean we'll be inconsistent.